src/components/reusables/Form.tsx

```diff
@@ -30,20 +32,16 @@ export default class Form extends React.Component<FormProps, {}> {
     event.preventDefault();
     let form = (this.refs["form"] as any);
     const data = new (window as any).FormData(form);
-    this.props.onSubmit(data).then(response => {
-      if (this.refs["successMessage"]) {
-        (this.refs["successMessage"] as HTMLElement).focus();
-      }
-    }).catch(err => {
-      if (this.refs["errorMessage"]) {
-        (this.refs["errorMessage"] as HTMLElement).focus();
-      }
-    });
+
+    await this.props.onSubmit(data);
```

Comment:
No need to use then/catch since the function declared is async.

Reply:
To make sure I understand this - using a single ref rather than two refs depending on the message type is unrelated to the new ref creation, it's just separate clean up?

Reply:
Correct, using a single ref is more cleanup.